packages/office-addin-project/src/convert.ts

```diff
@@ -47,25 +48,25 @@ async function asksForUserConfirmation(): Promise<boolean> {
 }
 
 async function backupProject(backupPath: string) {
-  const stream = fs.createWriteStream(backupPath);
-  const archive = archiver("zip", {
-    zlib: { level: 9 }, // Sets the compression level.
-  });
-
-  return new Promise<void>((resolve, reject) => {
-    archive
-      .glob(`{,!(node_modules)/**/}*`)
-      .on("error", (err) => reject(err))
-      .pipe(stream);
+  const zip: AdmZip = new AdmZip();
+  const outputPath = path.resolve(backupPath);
 
-    stream.on("close", () => {
-      console.log(
-        `A backup of your project was created to ${path.resolve(backupPath)}`
-      );
-      resolve();
-    });
-    archive.finalize();
+  const files: string[] = fs.readdirSync(__dirname);
+  files.forEach((entry) => {
```

Comment:
Does this work for multiple nested directories? I feel like this needed to be somehow recursive on the directories right?

Reply:
The adm-zip module automatically does nested folders when you call "addLocalFolder".  The reason for the loop is because I don't want every folder . . . this this reminds me that I forgot to add the code that skips some folders (i.e. node_modules)

Reply:
Looks like I didn't make it quite right in general.  I have now tested it better and the updated code should work.